bot/cogs/snakes.py

```diff
@@ -2,6 +2,11 @@
 import logging
 from typing import Any, Dict
 
+import discord
+import random
```

Comment:
This import should be two lines before `import discord`, since it's a standard library module.